### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by leggsimon**
/cc @adgad @VladDubrovskis @andrewgeorgiou1981

### CODE REVIEW

File: server/v1/controllers/concept.js [hunk 1]
```diff
@@ -60,8 +60,11 @@ module.exports = (req, res, next) => {
 			data.metadata.prefLabel = result.aggregations.tags.me.prefLabel.buckets[0].key;
 		}
 
+		const adUnitArray = [data.dfpSite]
```
Comment (adgad) [inline server/v1/controllers/concept.js]: Semicolons

File: server/v1/controllers/content.js [hunk 2]
```diff
@@ -35,8 +35,11 @@ module.exports = (req, res, next) => {
 
 			};
 
+			const adUnitArray = [dfpAttributes.dfpSite]
```
Comment (adgad) [inline server/v1/controllers/content.js]: Ditto

**Comment by adgad** [comment]
👍 + comments

File: server/v1/controllers/content.js (hunk 2 above)
Comment (VladDubrovskis) [inline server/v1/controllers/content.js]: Can this be a const? if we are changing it literally in the next line?

Comment (adgad) [inline server/v1/controllers/content.js]: You're allowed to push/modify the contents of the array, just not change the reference

Comment (leggsimon) [inline server/v1/controllers/content.js]: I was going for ES6's "reduced" syntactic sugar style 😝 

Will change

Comment (adgad) [inline server/v1/controllers/content.js]: Oh I meant it was fine

**Comment by adgad** [comment]
👍

**Pull request merged**